### ISSUE TITLE
Config.py file issues

### ISSUE THREAD

**Issue by rijafida** [cudeso/misp2sentinel#105]
![Screenshot 2024-09-11 130102](https://github.com/user-attachments/assets/4b0320eb-567b-42fa-a440-f37686fa6585)

We have tried implementing this guide but there is a lot of confusion regarding the flow of the steps. in the screenshot attached the 3rd point highlights adding value for tenant. What value is that / is it the code right below it with the values replaced to reflect our environment? 

Then the changes suggested to config.py file we assume here is the azure function. We need to make changes to the config.py file in the azure function?

Also do you have any documentation that deals with Graph method and Upload Indicator method separately. Everything is crammed into one article which makes it kind of hard to follow.   This would really help us out Thanks.

**Comment by cseelhoff-ms** [comment]
the example json in the readme did not work for me either. try this format instead: 

```
{
    "<TENANT_ID_WITH_APP_1>": {
      "id": "<APP_ID>",
      "secret": "<APP_SECRET>",
      "workspaceid": "<WORKSPACE_ID>"
    },
    "<TENANT_ID_WITH_APP_N>": {
      "id": "<APP_ID>",
      "secret": "<APP_SECRET_N>",
      "workspaceid": "<WORKSPACE_ID_N>"
    }
}
```

**Comment by rijafida** [comment]
We did try these changes based on some hit and trial and it worked ! 👍

**Comment by cudeso** [comment]
Updated example JSON with https://github.com/cudeso/misp2sentinel/commit/e57c84ab83d6177288ec41417fadbbe3df0ee285 based on feedback @cseelhoff-ms
